### DISCUSSION TITLE
Can support Chinese domestic database DM？

### DISCUSSION THREAD

**Discussion by 0x0bit** [mikro-orm/mikro-orm#5815] [Ideas]
**Is your feature request related to a problem? Please describe.**
Can support Chinese domestic database [DM](https://eco.dameng.com/document/dm/zh-cn/start/index.html)？

**Comment by B4nan** [comment]
I am not personally interested in that. In fact, I can't even open that link, it timeouts on my end. 

You can create your own drivers if you want.
